packages/core/src/codecatalyst/commands.ts

```diff
@@ -222,6 +231,9 @@ export class CodeCatalystCommands {
     }
 
     public createDevEnv(): Promise<void> {
+        if (vscode.env.remoteName === 'ssh-remote' && isInDevEnv()) {
```

Comment:
Do we only need `isInDevEnv()`?

Reply:
I think we also want to disable remote environments as well since create tries to open a devEnv and that's not possible on remote envs (as far as I know)